libqtile/scripts/check.py

```diff
@@ -95,9 +95,14 @@ def type_check_config_args(config_file):
     if shutil.which("mypy") is None:
         print("mypy not found, can't type check config file" "install it and try again")
         return
+    ver = sys.version_info
+    if (ver.major < 3) or (ver.major == 3 and ver.minor < 8):
```

Comment:
I think we can skip the `ver.major` check and just check `sys.version_info.minor`.

Reply:
Fair enough. The script will probably fail earlier on python2, and python4 probably won't release for a _long_ time.